wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+  group_by(investor_name) %>% 
+  summarise(total_holdings = n(), sum_value_usd = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup()
+datatable(investors_overview)
+```
+
+### Total holdings and value per investor and portfolio
+Number of holdings and aggregated value in USD of the portfolio input - grouped by investor and portfolio
+
+* **total_holdings** is the number of holdings per investor and portfolio
+* **sum_value_usd** is the aggregated value of the holdings in USD by investor and portfolio
+* **sum_market_value** .... see comment below
+* **pct_holdings** is the share of holdings of a specific PF, relative to the overall number of holdings in the analysis. It's the weight of the PF in pure terms of different holdings.
+* **value_weight** is the share of the value of a specific PF in USD, relative to the overall value across all PFs in the analysis. It shows how much value each of the PFs make up for
+
+**OPEN:**
+
+*sum_market_value* only makes sense if all holdings in a portfolio share the same currency. Otherwise, the overall value can look artificially inflated or deflated. Additionally grouping by currency would alleviate this, but the information on the PF level will not be readable anymore.
+
+```{r}
+investors_portfolio_overview <- result_data_qa %>% 
+  filter(investor_name != "Meta Investor") %>% 
+  group_by(investor_name, portfolio_name) %>% 
+  summarise(total_holdings = n(),
+            sum_value_usd = round(sum(value_usd, na.rm = T)),
+            sum_market_value = round(sum(market_value, na.rm = T))) %>% 
+  ungroup() %>% 
+  mutate(pct_holdings = round(total_holdings / sum(total_holdings),3),
+         value_weight = round(sum_value_usd / sum(sum_value_usd), 3))
+datatable(investors_portfolio_overview)
+
+investors_portfolio_currency_overview <- result_data_qa %>% 
+  filter(investor_name != "Meta Investor") %>% 
+  group_by(investor_name, portfolio_name, currency) %>% 
+  summarise(total_holdings = n(),
+            sum_value_usd = round(sum(value_usd, na.rm = T)),
+            sum_market_value = round(sum(market_value, na.rm = T))) %>% 
+  ungroup() %>% 
+  mutate(pct_holdings = round(total_holdings / sum(total_holdings),3),
+         market_value_weight = round(sum_market_value / sum(sum_market_value), 3))
+datatable(investors_portfolio_currency_overview)
+
+```
+
+
+## Currencies
+
+### currencies in portfolio and missing currency info
+1. Overview of all currencies present in the portfolio and their respective overall market values (in units of that currency)
+2. Overview of holdings with missing exchange rate information. For successful analysis, exchange rate information is required, so it should be checked if exchange rates can be manually added in these case.
+
+**Notes:**
+
+* Multiple entries of the same ISIN in the missing-table sometimes due to different holding IDs.
+* **market_value_miss_currency** is the aggregated value of the holdings with missing currency information per Investor and PF.
```

Comment:
This is great. Perhaps a statement as to whether there are any missing currencies